### ISSUE TITLE
Make the command line args a bit cleaner

### ISSUE THREAD

**Issue by gamozolabs** [gamozolabs/mesos#1]
Print out usage on `-h`, standardize usage formats, etc.

**Comment by marpie** [comment]
I don't know if it's me not knowing how to use a rust-tool or something else, but I needed to change the following lines to get Mesos running:

**File**: *main.rs* (Line: 54)
```rust
            if arg == "-p" {
                pid = Some(args[i+2].parse().unwrap());
            }
```

Old code: `pid = Some(arg.parse().unwrap());`

**Comment by gamozolabs** [comment]
Fixed in 8d4ae430385920b371be76c19dc81b68d194caa1 . Sorry, current users were using the process spawning option and it looks like the PID version did not get tested.
